tools/test/test_actions_local_runner.py

```diff
@@ -85,19 +85,18 @@ def test_lint(self):
             for line in self.expected:
                 self.assertIn(line, stdout)
 
-            self.assertIn("✓ mypy", stdout)
+            self.assertIn("mypy", stdout)
 
         def test_quicklint(self):
             cmd = ["make", "quicklint", "-j", str(multiprocessing.cpu_count())]
             proc = subprocess.run(cmd, cwd=actions_local_runner.REPO_ROOT, stdout=subprocess.PIPE)
             stdout = proc.stdout.decode()
 
-            # TODO: See https://github.com/pytorch/pytorch/issues/58253
-            # for line in self.expected:
-            #     self.assertIn(line, stdout)
+            for line in self.expected:
```

Comment:
The issue is closed and doesn't really matter now that we don't check for passing runs